src/app/containers/ATIAnalytics/index.jsx

```diff
@@ -5,24 +5,24 @@ import CanonicalATIAnalytics from './canonical';
 import AmpATIAnalytics from './amp';
 import ArticleAtiParams from './params/article';
 import FrontPageAtiParams from './params/frontpage';
+import MediaPageAtiParams from './params/media';
 import { articleDataPropTypes } from '../../models/propTypes/article';
 import { frontPageDataPropTypes } from '../../models/propTypes/frontPage';
 
 const ATIAnalytics = ({ data }) => {
   const { pageType, platform } = React.useContext(RequestContext);
 
-  let pageviewParams = '';
-  switch (pageType) {
-    case 'article':
-      pageviewParams = ArticleAtiParams(data);
-      break;
-    case 'frontPage':
-      pageviewParams = FrontPageAtiParams(data);
-      break;
-    default:
-      return null;
+  const validPageTypes = ['article', 'frontPage', 'media'];
+  if (!validPageTypes.includes(pageType)) {
+    return null;
   }
 
+  const pageviewParams = {
```

Comment:
This object has the same keys as the array on line 15, can you use `Object.keys(pageviewParams)` to save duplication?

Reply:
@dr3 do you mean like this?

Reply:
@dr3 updated it - do you mean like this?